source/Calamari/Commands/ReleasePackageLockCommand.cs

```diff
@@ -56,6 +50,8 @@ public override int Execute(string[] commandLineArguments)
                 return ConsoleFormatter.PrintError(log, ex);
             }
 
+            packageJournal.ExpireStaleLocks(TimeSpan.FromDays(14));
```

Comment:
What time-frame is reasonable for a lock to be marked as stale? Should this be a variable?